HarrisMoviesApplication/TableViewProtocols.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  TableViewProtocols.swift
+//  HarrisMoviesApplication
+//
+//  Created by juan harrington on 24/03/22.
+//
+
+import UIKit
+
+
+extension ViewController: UITableViewDelegate, UITableViewDataSource{
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return factsArray.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell =  catFactTableView.dequeueReusableCell(withIdentifier: "prototypeCell", for: indexPath)
+        //cell.textLabel?.text = vamo[indexPath.row] esto es lo que hay que arreglar pa
```

Comment:
What is "vamo" ?

Reply:
was a testing name for a variable i already resolved it